src/c++/perf_analyzer/concurrency_manager.cc

```diff
@@ -145,6 +145,8 @@ ConcurrencyManager::ReconfigThreads(const size_t concurrent_request_count)
         active_threads_++;
       }
     }
+    // TODO REFACTOR TMA-1043 the memory manager should have API to set
```

Comment:
Why is this todo here?

Reply:
This is where active_threads_ is changed and is the real location that someone/something should be notified that it changed. As it stands, we have to go from Manager to all Workers to all their Contexts to communicate that this value has changed, and it is only used in one place.